testsupport/user_assertions.go

```diff
@@ -119,3 +122,35 @@ func GetMurTargetMember(t *testing.T, awaitilities wait.Awaitilities, mur *toolc
 	require.FailNowf(t, "Unable to find a target member cluster", "MasterUserRecord: %+v", mur)
 	return nil
 }
+
+func DeletedRoleAndAwaitRecreation(t *testing.T, memberAwait *wait.MemberAwaitility, ns corev1.Namespace, role string) {
+	userRole, err := memberAwait.WaitForRole(&ns, role)
+	require.NoError(t, err)
+	require.NotEmpty(t, userRole)
+	require.Contains(t, userRole.Labels, "toolchain.dev.openshift.com/owner")
```

Comment:
I guess that this check could be moved to "namespaceObjectChecks" in https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/tiers/checks.go
https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/tiers/checks.go#L358-L466

Reply:
sure, thank you for pointing that out. I wasn't super sure where to move it, so not my best decision to move it to user_assertions

Reply:
actually on second thought, since the function deletes it and waits for recreation, Not sure if it is a check?

Reply:
I meant just the single line:
```
 	require.Contains(t, userRole.Labels, "toolchain.dev.openshift.com/owner") 
```
because we expect that all the created objects should contain the label, right?

Reply:
oh sure, i misunderstood.

Reply:
@MatousJobanek done in 3723235